home-service/src/main/resources/application-home-service.properties

```diff
@@ -0,0 +1,12 @@
+#Mail service
+spring.mail.host=smtp.gmail.com
+spring.mail.port=587
+spring.mail.username=${MAIL_USERNAME:test.ita.emails@gmail.com}
+spring.mail.password=${MAIL_PASSWORD:itacademy2021}
+spring.mail.properties.mail.smtp.auth=true
+spring.mail.properties.mail.smtp.starttls.enable=true
+
+#Quartz service
+spring.quartz.job-store-type = jdbc
+spring.quartz.properties.org.quartz.jobStore.class=org.quartz.impl.jdbcjobstore.JobStoreCMT
+spring.quartz.properties.org.quartz.jobStore.driverDelegateClass=org.quartz.impl.jdbcjobstore.PostgreSQLDelegate
```

Comment:
I propose to rename this file to application.properties according to general naming of such files in the project